packages/collector/test/tracing/logger/instana-logger/controls.js

```diff
@@ -19,20 +17,18 @@ const agentControls = require('../../../globalAgent').instance;
 let appProcess;
 let appPort;
 
-// TODO: transform into class
-
 exports.registerTestHooks = (opts = {}) => {
   let appName = 'app.js';
   if (opts.instanaLoggingMode) {
     switch (opts.instanaLoggingMode) {
-      case 'instana-creates-bunyan-logger':
-        appName = 'app-instana-creates-bunyan-logger.js';
+      case 'instana-creates-pino-logger':
```

Comment:
suggestion: you can also use generic names such as "instana-uses-default-logger" and "instana-uses-custom-pino-logger" etc. With this, we never have to change the filenames or strings when going away from pino.